.github/workflows/release.yml

```
@@ -0,0 +1,166 @@
+            --exclude=node_modules \
+            --exclude=.git \
+            --exclude=logs \
+            --exclude=**/*.log \
+            .
+            
+      - name: Generate changelog
+        id: changelog
+        run: |
+          # Get the previous tag
+          PREV_TAG=$(git describe --tags --abbrev=0 HEAD^ 2>/dev/null || echo "")
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)